src/jobs/models/kubernetes/jobConnector.js

```diff
@@ -84,7 +84,8 @@ module.exports.deleteAllContainers = async (jobPlatformName) => {
             deleted++;
         }
     }
-    return { deleted };
+    const jobExperimentsDeleted = jobExperimentHandler.clearAllFinishedJobExperiments();
+    return { deleted, internal_resources_deleted: jobExperimentsDeleted };
```

Comment:
internal_resources_deleted:  I think we should open it to expansion and have it as an object (maybe in the future we'll have other internal resources)

like
```
internal_resources_deleted: {
 chaos-mesh: x
}
```

Reply:
done.